client/src/test/java/com/tigrisdata/db/client/model/ReadFieldsTest.java

```diff
@@ -0,0 +1,29 @@
+package com.tigrisdata.db.client.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReadFieldsTest {
```

Comment:
nit, can you add test what should be done to include all?

Reply:
added test.